### ISSUE TITLE
model error

### ISSUE THREAD

**Issue by Iciir** [bonastreyair/node-red-contrib-teachable-machine#35]
I am suddenly getting this error:

![image](https://user-images.githubusercontent.com/93911948/143627524-dc0312cd-d3ab-4e5e-a54e-9df158eec298.png)

and since I tried relading it:

![image](https://user-images.githubusercontent.com/93911948/143627557-7e437d14-9386-4758-a1e1-6e825c2314a5.png)

![image](https://user-images.githubusercontent.com/93911948/143627583-171cc855-4042-4384-9339-faa9947be06f.png)

Any idea what could cause this?  It worked fine until about 1-2 days ago.  I noticed the error but ignored it because I was busy, now I see reloading doesn't fix it, the URL is correct.  So.... what upset him suddenly?  :(

**Comment by bonastreyair** [comment]
That is really weird, never happened to me before.
 - If you reboot node-red it still happens? 
 - If you recreate the node it still happens?
 - If you use a new trained model it still happens?

The part from `model is not ready`, is expected if the model is not loaded and you are already sending an image.
What is you image sending rate? I recommend putting a `delay` node in front of the `teachable machine node` to limit the speed of images that will be really processed, if you are sending them really fast, you could end up with queued images to be processed and there is no limit, so you could theoretically filling your entire RAM.

I'll need more information regarding this problem to be able to reproduce it and fix it in case it is really a bug from the node. Thanks!

**Comment by Iciir** [comment]
❌ rebooting Node-RED
❌ recreating the node
❌ using a new trained model

Actually I am quite happy that it still doesn't work with a different project, because I did work on the first one and I wouldn't have wanted it to be "too big" or something.

Do you know of any way in which we could debug what is happening?

I can also say, I did not install anything new in Node-RED, only changes to node arrangements.

Maybe a... system update, broke something?  Is that possible?  I mean a simple `sudo apt upgrade`.

**Comment by Iciir** [comment]
I tried uninstalling and reinstalling `node-red-contrib-teachable-machine`, still the same error.  Hmm....

**Comment by bonastreyair** [comment]
Did you try in a Docker? I have no problems with a clean installation.
It feels like some dependency from Node-RED has totally broken.

**Comment by Iciir** [comment]
I just run it installed through `npm`, I actually don't know what docker is, didn't have to know yet. 

I tried it in a Virtual Machine with a fresh install and it's working perfectly so yes...
> some dependency from Node-RED has totally broken.

Now, how do I find out how to fix this because I am quite afraid of problems when porting everything I've done to a fresh install.

Maybe I should backup everything and do a `npm audit fix --force`. This will take a day or two to recover from, let's see.  Without --force, doesn't solve it. 

EDIT:  Actually... this might give a hint:

```
node-fetch  <2.6.1
The `size` option isn't honored after following a redirect in node-fetch - https://github.com/advisories/GHSA-w7rc-rwvf-8q5r
fix available via `npm audit fix --force`
Will install node-red-contrib-teachable-machine@1.1.2, which is a breaking change
```

**Comment by Iciir** [comment]
I got an idea to travel here, and found this:

![image](https://user-images.githubusercontent.com/93911948/143988346-1b904499-259d-465f-ac6f-01ce4370e565.png)

**Comment by Iciir** [comment]
Then I went to `~/node-red/node_modules/node-red-contrib-teachable-machine/` and fixed everything in there, everything is fixed, then I completely sudo uninstalled `node-red-contrib-teachable-machine`, deleted the folder, reinstalled through Node-RED Palettes, uninstalled through there, reinstalled through there, `npm audit` shows zero vulnerabilities.

Still, same error.

I'll keep trying a bit more, to maybe help future users.

**Comment by bonastreyair** [comment]
I recommend to reinstall Node-RED or use a Docker with Node-RED.

I'll close this issue since tested using a clean install there are no errors. Thanks for the information it might help future newcomers.

**Comment by Iciir** [comment]
I found the problem and solved it!

`npm-audit fix --force`, reinstalling Node-RED fresh or anything like that has **nothing** to do with it, I just broke everything with those type of things, and had to revert to a backup.

The problem was:  `node-red-contrib-tesseract`.  That's all, **removed that from Palettes** and the model loads without problem.  

![image](https://user-images.githubusercontent.com/93911948/144851535-3b10c5f8-d52a-44f5-9a62-02711b140288.png)

Problem node:

![image](https://user-images.githubusercontent.com/93911948/144851682-e0e2f25b-242e-4957-a6bf-e01116d6ca55.png)
